### ISSUE TITLE
Raw latex in yaml headers

### ISSUE THREAD

**Issue by trymnf** [DCsunset/pandoc-include#6]
I have an issue when using raw Latex in the document's yaml header: 

```
---
title: Title
date: \today{}
---

Here is a file in the same directory:

!include file.md 
```
Yields the following pdf: 
 
![image](https://user-images.githubusercontent.com/47108592/69162893-9ade2c80-0aed-11ea-949e-f69cd07d4e38.png)

This is not a huge problem on its own, but `header-includes` also doesn't work, which is a bit more of an issue. 

Any thoughts?

**Comment by DCsunset** [comment]
After inspecting it for several days, I found that the assignment of metadata was problematic.
It has been fixed in v0.6.2

**Comment by DCsunset** [comment]
69b60e1dfd54f08e1bc5fbc122252965eb7bc0bd
